sabnzbd/deobfuscate_filenames.py

```diff
@@ -118,11 +118,21 @@ def is_probably_obfuscated(myinputfilename):
     return True  # default not obfuscated
 
 
-def deobfuscate(workingdirectory, usefulname):
-    """ In workingdirectory, check all filenames, and if wanted, rename """
+def deobfuscate_dir(workingdirectory, usefulname):
+    """ In workingdirectory, check all filenames, and if wanted, deobfuscate """
+
+    list_of_files = []
+    for (dirpath, dirnames, filenames) in os.walk(workingdirectory):
+        list_of_files += [os.path.join(dirpath, file) for file in filenames]
+    deobfuscate_list(list_of_files, usefulname)
+
+
+def deobfuscate_list(filelist, usefulname):
```

Comment:
Might just have 1 function `deobfuscate` since it's so easy to use `globber_full` as the input in case a directory listing needs to be used.

Reply:
Indeed, I had already done that.